### PR TITLE
p7zip: add page

### PR DESCRIPTION
**Pull request by mt-empty**
- [x ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/p7zip.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# p7zip
+
+> Wrapper on 7-Zip file archiver with high compression ratio.
```
Comment (zdroid) [inline pages/common/p7zip.md]: Should it be `Wrapper of 7-Zip ...` instead?

**Comment by zdroid** [comment]
You should make a commit with your GitHub account and sign the CLA.

File: pages/common/p7zip.md (hunk 1 above)
Comment (mt-empty) [reply pages/common/p7zip.md]: line 3 was copied from the man pages. but I will go ahead and make the changes.

File: pages/common/p7zip.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# p7zip
+
+> Wrapper of 7-Zip file archiver with high compression ratio.
+> Internally executes either 7za or 7zr command.
+
+- Archive a file, replacing it with a 7zipped compressed version:
+
+`p7zip {{path/to/file}}`
+
+- Archive a file without removing the input file:
```
Comment (andrik) [inline pages/common/p7zip.md]:
```suggestion
- Archive a file keeping the input file:
```

File: pages/common/p7zip.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# p7zip
+
+> Wrapper of 7-Zip file archiver with high compression ratio.
+> Internally executes either 7za or 7zr command.
+
+- Archive a file, replacing it with a 7zipped compressed version:
```
Comment (andrik) [inline pages/common/p7zip.md]:
```suggestion
- Archive a file, replacing it with a compressed version:
```

File: pages/common/p7zip.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Archive a file without removing the input file:
+
+`p7zip -k {{path/to/file}}`
+
+- Decompress a file, replacing it with the original uncompressed version:
+
+`p7zip -d {{compressed.ext}}.7z`
+
+- Decompress a file without removing the input file:
```
Comment (andrik) [inline pages/common/p7zip.md]:
```suggestion
- Decompress a file keeping the input file:
```

**Comment by mt-empty** [comment]
I think I incorrectly cloned the repo, need to change personas.

**Pull request closed without merging**